R/transform.R

```diff
@@ -4,11 +4,17 @@
 #'   the result.
 #' @param files A character vector with paths to the file that should be
 #'   transformed.
+#' @param assert_code Boolean to indicate whether checking for parsing errors
+#'   should be done prior to styling.
 #' @inheritParams make_transformer
 #' @return A logical value that indicates whether or not any file was changed is
 #'   returned invisibly. If files were changed, the user is informed to
 #'   carefully inspect the changes via a message sent to the console.
-transform_files <- function(files, transformers) {
+transform_files <- function(files, transformers, assert_code = TRUE) {
```

Comment:
Yes, with `any(changed, na.rm = TRUE)`, we only get the first warning message. So I will undo my error handling implementation and adapt the tests.